otomi-install/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: otomi-install
```

Comment:
I don't know about the others but I would just name it otomi. I think it's standard if I look at the naming of other open source charts (e.g. https://github.com/prometheus-community/helm-charts/tree/main/charts: alertmanager is just called alertmanager, prometheus just prometheus).

Reply:
yes please

Reply:
and can this folder be renamed "chart" so it is clear it is the chart that comes with this software/